variables.tf

```diff
@@ -79,8 +79,9 @@ variable "vpc_regions" {
   EOD
 
   type = map(object({
-    vpcaccess = bool
-    nat       = number
+    vpcaccess            = bool
+    nat                  = number
+    nat_min_ports_per_vm = optional(number)
```

Comment:
Would you be so kind and also add this key to the description of the `europe-west1` region?

Reply:
If `nat_min_ports_per_vm` is set for one object, it has to be for all others. Otherwise, Terraform will throw a `all map elements must have the same type` error. I don't believe this can be overcome with your current object structure. It won't break existing config if not set.
You still want the config to be part of the example? Again, that'd mean for all the regions.